README.md

```diff
@@ -4,11 +4,20 @@
 
 ### Disclaimer: work in progress and not ready for production use. 
 
-kube-bind is a prototype project with the goal to establish a new extension model for Kubernetes clusters:
+You are invited to [contribute](#contributing)!
 
-- APIs should be bindable into a cluster and operated by a service provider
-- these APIs should not require (custom) controllers/operators run locally in the consuming cluster
-- only a single vendor-neutral, OpenSource agent should be required.
+## What is it?
+
+kube-bind is a prototype project that aims to provide better support for service providers and consumers that reside in distinct Kubernetes clusters.
+
+- A service provider defines its API in terms of CRDs and associated permission claims/limitations, and exports it for use from other clusters.
+- Service consumers identify the services they want to consume.
```

Comment:
What about this? 
`Consumers choose the service they want to consume and sent a bind request through the service provider exposed API`

Reply:
Is this accurate?  I thought that the service consumer creates a local API object declaring the desire to consume a service.

Reply:
Here API means HTTP API.